.packit.yaml

```diff
@@ -5,15 +5,18 @@ synced_files:
 upstream_package_name: fmf
 downstream_package_name: fmf
 
+# Epel9 fails to build with dynamic version. Need to create archive with PKG-INFO
```

Comment:
I don't remember if we investigated it here, `tmt` or `packit`, but I did confirm this issue when I've tried to make a PR to remove it once. What about we go ahead with this approach for now since we know it works and open a PR to keep track if this changes. Maybe there was a package bump that fixed it. Should note that this is only affecting upstream `copr_build` so it's not as critical as the sync part.

> The issue I can see is %autorelease macro, which doesn't work on el9. Maybe it would be good to have a sed there somewhere and replace it with 1.

This actually works a-ok :)

Reply:
> This actually works a-ok :)
autorelease works on el9? Am I missing some package?

Reply:
> I did confirm this issue

Maybe it detects the warning from setuptools_scm thinking it uses incompatible setuptools version, even though we're using hatchling (dependency hell).  
I'm thinking the warning could be solved by explicitly setting setuptools_scm version, but haven't tested it yet.

Reply:
> autorelease works on el9? Am I missing some package?

Well, some rust packages are packaged to epel9 and they use `%autorelease` ([example](https://src.fedoraproject.org/rpms/rust-libsqlite3-sys/blob/rawhide/f/rust-libsqlite3-sys.spec)). I didn't got through the koji builds to confirm that it works as intended though.

> Maybe it detects the warning from setuptools_scm thinking it uses incompatible setuptools version, even though we're using hatchling (dependency hell).

It was not a warning, it was erroring that metadata was not packaged, maybe I will push a temporary commit to check this again since I don't need to `/packit build` manually here to run things.

Reply:
Ok, check one of the recent runs for EPEL9, the issue is still reproducible